src/pip/_internal/vcs/bazaar.py

```diff
@@ -40,13 +39,12 @@ def export(self, location):
         if os.path.exists(location):
             rmtree(location)
 
-        with TempDirectory(kind="export") as temp_dir:
-            self.unpack(temp_dir.path)
-
-            self.run_command(
-                ['export', location],
-                cwd=temp_dir.path, show_stdout=False,
-            )
+        url, rev = self.get_url_rev()
+        rev_options = self.make_rev_options(rev)
```

Comment:
FYI, with what was just committed to master, you'll want to replace these two lines with the single line:

```python
url, rev_options = self.get_url_rev_options()
```